spec/models/category_spec.rb

```diff
@@ -2,6 +2,40 @@
 
 
 describe Category do
-
   it {should have_many(:videos)}
-end
\ No newline at end of file
+  describe "#recent_videos" do
+
+    it "returns empty array if category has no videos" do
+      comedy = Category.create(name: 'Comedy') 
+      expect(comedy.recent_videos).to eq([])
+    end
+
+    it "returns array in reverse chronological order" do
+      action = Category.create(name: 'Action') 
+      wop_gun = Video.create(title: 'Wop Gun', description: 'Action Flick', categories: [action],created_at: 3.day.ago)
+      vop_gun = Video.create(title: 'Vop Gun', description: 'Action Flick', categories: [action],created_at: 2.day.ago)
+      pop_gun = Video.create(title: 'Pop Gun', description: 'Action Flick', categories: [action],created_at: 1.day.ago)
+      expect(action.recent_videos).to eq([pop_gun,vop_gun,wop_gun])
```

Comment:
`[pop_gun,vop_gun,wop_gun]` this looks very stacked. Why there are no spaces in between? :)